README.md

```diff
@@ -238,14 +239,14 @@ splunk_conf 'system/test.conf' do
 end
 ```
 
-###### Stanza-level Evaluation
+###### Section-level Evaluation
 
-Stanza-level evaluation allows you to evaluate the config key values for a specific stanza without being
+section-level evaluation allows you to evaluate the config key values for a specific section without being
 responsible for regurgitating all of the config you didn't want to modify.
 
-In this example, only a specific stanza's contents are turned into lowercase strings. The rest is
-merged as usual. **Note that the Stanza-level proc receives the section string and a props hash, and expects
-a pair (array) of the desired section string and props hash.**
+In this example, only a specific section's contents are turned into lowercase strings. The rest is
+merged as usual. **Note that the section-level proc receives the context and a props hash, and expects
+a pair (array) of the desired section name and props hash to be returned.**
```

Comment:
Out of curiosity, what's the usecase for letting the user change the section name dynamically with a Section level stanza? (And how does it work if I have a hash with one section with the name that it targets dynamically already set?)